burn-tch/build.rs

```diff
@@ -0,0 +1,15 @@
+use std::env;
+fn main() {
```

Comment:
This is so small, but I would like spaces between imports and functions 😅. I guess the `build.rs` is not formatted by `cargo fmt` ?

Reply:
OK. Not sure why there is no space.